helm/cluster-api-provider-aws/Chart.yaml

```
@@ -3,4 +3,4 @@ name: cluster-api-provider-aws
 description: "Helm chart for cluster-api-provider-aws."
 home: "https://github.com/giantswarm/cluster-api-provider-aws-app"
 version: "[[ .Version ]]"
-appVersion: "0.6.6-gs2"
+appVersion: [[ .AppVersion ]]
```

Maybe this needs in between "", not sure

```suggestion
appVersion: "[[ .AppVersion ]]"
```

---

I remember those parentheses hunting us a while ago. But I think it was mainly an issue when we used it for the watch filter label.